easy-deploy-script/easy-deploy-viral-ngs.sh

```diff
@@ -92,6 +92,20 @@ if [ $? -ne 0 ]; then
     fi
 fi
 
+
+# Usage: puniq [<path>]
+# Remove duplicate entries from a PATH style value while retaining
+# the original order. Use PATH if no <path> is given.
+#
+# Example:
+#   $ puniq /usr/bin:/usr/local/bin:/usr/bin
+#   /usr/bin:/usr/local/bin
+puniq() {
```

Comment:
We should use this for `prepend_miniconda` as well.

Reply:
Done